src/core/data/middlewares/attach-mock/index.ts

```diff
@@ -57,133 +66,201 @@ export async function attachMock(this: Provider, params: MiddlewareParams): Prom
 	}
 
 	const
-		{opts, ctx} = params;
+		{opts, ctx} = params,
+		mocks = await getProviderMocks(this, opts);
+
+	if (!mocks) {
```

Comment:
Если значение не логическое, то лучше явно прописывать условие (выше декларативность)

```suggestion
	if (mocks == null) {
```